axum-extra/src/extract/host.rs

```diff
@@ -148,18 +167,55 @@ mod tests {
     async fn ip4_uri_host() {
         let mut parts = Request::new(()).into_parts().0;
         parts.uri = "https://127.0.0.1:1234/image.jpg".parse().unwrap();
-        let host = Host::from_request_parts(&mut parts, &()).await.unwrap();
+        let host =
+            <Host as axum::extract::FromRequestParts<_>>::from_request_parts(&mut parts, &())
+                .await
+                .unwrap();
```

Comment:
```suggestion
        let host = parts.extract::<Host>().await.unwrap();
```

I think this should work too if the `RequestPartsExt` trait is imported